prog/dftb+/lib_reks/reksio.F90

```diff
@@ -180,18 +180,20 @@ subroutine printReksGradInfo(this, derivs)
           end if
         end do
 
-!        write(stdOut,*) "AVG state"
-!        write(stdOut,'(3(f15.8))') this%avgGrad(:,:)
-!        write(stdOut,'(3(f15.8))')
-!        do ist = 1, this%nstates
-!          write(stdOut,*) ist, "st state (SA-REKS)"
-!          write(stdOut,'(3(f15.8))') this%SAgrad(:,:,ist)
-!          if (ist == this%nstates) then
-!            write(stdOut,"(A)") repeat("-", 50)
-!          else
-!            write(stdOut,'(3(f15.8))')
-!          end if
-!        end do
+        if (this%Plevel >= 2) then
+          write(stdOut,"(12X,A)") "AVG state"
```

Comment:
Any reason not to write AVG in full instead of as an acronym?

Reply:
Done.